doc/releases/changelog-dev.md

```diff
@@ -165,6 +165,19 @@
   and the codecov check itself would never execute.
   [(#5101)](https://github.com/PennyLaneAI/pennylane/pull/5101)
 
+* String representations of Pauli operators have been improved and there are new aliases `X, Y, Z, I` for `PauliX, PauliY, PauliZ, Identity`.
+  ```
+  >>> qml.PauliX(0)
+  X(0)
+  >>> qml.PauliX('a')
+  X('a')
+  >>> 0.5 * X(0) # no unnecessary bracket
```

Comment:
Minor point but the comments on the unnecessary bracket feel harder to understand if you don't know the "before" version.

Reply:
So just not make that comment and say "here is how (nice) it looks now" 🙂 ?